### DISCUSSION TITLE
Conflicto de color de los componentes que muestran gastos extras en la marca Blau

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1134] [Proposals]
### Goal
Necesitamos definir como solucionar el problema de globalidad de los componentes que usan el color Highligh, ya que en Blau el highligh es azul y pierde el tono rojizo que se utiliza para informar de gastos extras

### Context
Hay algunos módulos en CPS que necesitan generar una excepción de color en la marca Blau y puesto que se está utilizando una "if" para esto, desde CPS nos piden una Mistificación con una nueva variable para dar solución a esto.

### Impact
Esto es necesario de cara a mantener la globalidad de los componentes y evitar "ifs" en CPS

### Discussion Questions
¿Es necesario crear nueva variable llamada "extra_charges" para poder dar a cada marca una variable de color distinta ?
¿Cabe considerarse la opción cambiar el color highlight en todas las marcas a un error?

### Do you have a visual proposal?

https://www.figma.com/file/xNUwhH29z3Fre4foNltgtX/CPS_Specs?node-id=8537-137500

**Comment by isaacperezestrada-telefonica** [comment]
Related to https://jira.tid.es/browse/DSNCORE-1327

**Reply by yceballost** [reply]
ups, I didn't see this issue, we don't use Jira anymore. We manage all the tasks with Github for a long time ago.

**Comment by yceballost** [comment]
Agree con que el highlight debe ser un color para destacar, usar el mismo azul de marca Blau para el highlight no es lo óptimo porque no estariamos cubriendo el objetivo de este color.

Veo dos posibilidades
1. La que comentas, si consideráis que eso debe verse como un error, podéis cambiar a error.
2. Cambiar el color que está aplicando al highlight en Blau

Creo que lo ideal es ir por la segunda, no usar color de errores para cosas que NO son errores y buscar un color para Blau que cumpla esa función de destaque.

Tenéis alguna propuesta?
Os dejo por aquí un archivo donde se recogen los colores de la paleta de Blau por si os encaja alguno. A vote pronto el `signal_3`? `#8F133F` 😬

https://telefonicacorp.sharepoint.com/:b:/s/Colabora_TCX/EchNhWMRKyBAnnRf3kx10vcBMd5FwLwioU1gPylg4vw2-g?e=zcapnr

**Reply by MontseTelefonica** [reply]
Ok, tras revisar la paleta, veo que los únicos colores distintos al azul que podríamos usar como highlight :
-brand 3
-signal 3

¿Qué opinas @yceballost ?
<img width="209" alt="image" src="https://user-images.githubusercontent.com/95479574/226551958-39817e90-ac89-441a-8975-0fddd6c06fdc.png">

**Reply by yceballost** [reply]
el problema del brand_3 es que se usa en los links.. entonces igual puede llevar a confusión si usamos el mismo color para los highlights.

yo me quedaría con el signal_3 por ser un color muy diferente al resto, "en la línea" del resto de marcas, pero por poder, puede ser cualquiera que consideréis que funcione.

Otra opción es el brand_2 que se usa en los botones, yo por la misma razón del link evitaría asociar colores interactivos a un texto destacado que no lo es.

Otro camino es buscar una solución visual que no use color en el texto para destacar ese dato, quizás esta sea la clave (?)

**Reply by MontseTelefonica** [reply]
Por coste-beneficio optaría por usar el Signal 3 para el highlight :) mientras se puede valorar si hay otra opción me destacar el gasto extra sin usar color.

**Reply by yceballost** [reply]
https://github.com/Telefonica/mistica-design/issues/1141
